model/testing/src/icon4py/model/testing/serialbox.py

```diff
@@ -312,31 +312,48 @@ def edge_cell_length(self):
         return self._get_field("edge_cell_length", dims.EdgeDim, dims.E2CDim)
 
     def cells_start_index(self):
-        return self._read_int32_shift1("c_start_index")
+        start_idx = self._read_int32("c_start_index")
+        return np.where(start_idx == 0, start_idx, start_idx - 1)
 
     def cells_end_index(self):
         return self._read_int32("c_end_index")
 
     def vertex_start_index(self):
-        return self._read_int32_shift1("v_start_index")
+        start_idx = self._read_int32("v_start_index")
```

Comment:
ICON sets some of these values to 0. I suspect these are values that are not used otherwise ICON would crash since it has 1 based fields (?)/ But this might be an issue in the wrapper.